src/traces/funnelarea/attributes.js

```diff
@@ -0,0 +1,109 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var pieAttrs = require('../pie/attributes');
+var plotAttrs = require('../../plots/attributes');
+var domainAttrs = require('../../plots/domain').attributes;
+var hovertemplateAttrs = require('../../components/fx/hovertemplate_attributes');
+
+var extendFlat = require('../../lib/extend').extendFlat;
+
+module.exports = {
+    labels: pieAttrs.labels,
+    // equivalent of x0 and dx, if label is missing
+    label0: pieAttrs.label0,
+    dlabel: pieAttrs.dlabel,
+    values: pieAttrs.values,
+
+    marker: {
+        colors: pieAttrs.marker.colors,
+        line: {
+            color: extendFlat({}, pieAttrs.marker.line.color, {
+                dflt: null,
+                description: [
+                    'Sets the color of the line enclosing each sector.',
+                    'Defaults to the `paper_bgcolor` value.'
+                ].join(' ')
+            }),
+            width: extendFlat({}, pieAttrs.marker.line.width, {dflt: 1}),
```

Comment:
I'm having second thoughts about this today. :thinking: 

I think I would prefer making the dflt `marker.line.width` 0 (like pie)

![image](https://user-images.githubusercontent.com/6675409/58108074-24ea1e80-7bb9-11e9-9a56-e80b53a2ffda.png)


vs:

![image](https://user-images.githubusercontent.com/6675409/58108085-2ddaf000-7bb9-11e9-8a94-ed600eb85149.png)

Any strong opinions from @plotly/plotly_js and/or @nicolaskruchten ?

Reply:
No comments in ~24h (meaning no strong opinions, right :smirk: ), so I'll let @archmoj decide what he thinks looks best for `funnelarea`.

Reply:
I also like the integrity in the `pie` version i.e. with no white lines.
But since the edges between two colour may not be rendered perfectly without gap, I suggest we keep them (by the way i.e. similar to `sunburst`). In addition it may help the figures be clear when printing a report document in black & white.